goprocess/gp_test.go

```diff
@@ -1,9 +1,93 @@
 package goprocess
 
-import "testing"
+import (
+	"reflect"
+	"sort"
+	"testing"
+
+	"github.com/keybase/go-ps"
+)
 
 func BenchmarkFindAll(b *testing.B) {
 	for ii := 0; ii < b.N; ii++ {
 		_ = FindAll()
 	}
 }
+
+type fixtureFunc func() (pss []ps.Process, want []P, fn isGoFunc, done func())
```

Comment:
Nit: this is now unused an can be dropped as well.

Reply:
Done